pkg/server/handler.go

```diff
@@ -67,37 +67,19 @@ func init() {
 	)
 }
 
-const (
-	passthroughHeader   = "X-Kcp-Api-V1-Discovery-Passthrough"
-	workspaceAnnotation = "tenancy.kcp.dev/workspace"
-)
-
-type (
-	acceptHeaderContextKeyType int
-	userAgentContextKeyType    int
-)
-
-const (
-	// clusterKey is the context key for the request namespace.
-	acceptHeaderContextKey acceptHeaderContextKeyType = iota
-
-	// userAgentContextKey is the context key for the request user-agent.
-	userAgentContextKey userAgentContextKeyType = iota
-)
-
 // WithAcceptHeader makes the Accept header available for code in the handler chain. It is needed for
 // Wildcard requests, when finding the CRD with a common schema. For PartialObjectMeta requests we cand
 // weaken the schema requirement and allow different schemas across workspaces.
 func WithAcceptHeader(apiHandler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		ctx := context.WithValue(req.Context(), acceptHeaderContextKey, req.Header.Get("Accept"))
+		ctx := context.WithValue(req.Context(), AcceptHeaderContextKey, req.Header.Get("Accept"))
```

Comment:
I don't think we want to leak this.

Reply:
I guess it doesn't matter. When I tried to move `Config` into its own package, the same way `Options` is, I had to expose it. I gave up on that for now, so we can make it private again.